pkg/configuration/configuration.go

```diff
@@ -93,13 +93,20 @@ const (
 	// varDeactivationDomainsExcluded is a string of comma-separated domains that should be excluded from automatic user deactivation
 	// For example: "@redhat.com,@ibm.com"
 	varDeactivationDomainsExcluded = "deactivation.domains.excluded"
+
+	// varForbiddenUsernamePrefixes defines the prefixes that a username may not have when signing up.  If a
+	// username has a forbidden prefix, then the username compliance prefix is added to the username
+	varForbiddenUsernamePrefixes = "username.forbidden.prefixes"
+
+	DefaultForbiddenUsernamePrefixes = "openshift,kubernetes"
```

Comment:
there are namespaces prefixed with `kube-`, we should include it in the list of forbidden prefixes

Reply:
Done.

Reply:
I can't see the change in your PR 😳 
Also, would it be simpler if the var was an `[]string` instead of a list of value joined with commas?

Reply:
Here's the line:

https://github.com/codeready-toolchain/host-operator/pull/325/files#diff-a0e28739a3c85fb2099ec7cdf95f0579f5f84488cca12aee25fb2e62702db5d6R101

Arrays can't be constants in Go, so we need the comma-separated string.